src/main/java/lotto/view/InputView.java

```diff
@@ -38,9 +57,14 @@ public WinningNumbers inputWinningNumbers() {
 
         int bonusNumber = scanner.nextInt();
 
-        return new WinningNumbers(new LottoNumbers(Arrays.stream(numbers)
-                    .map(Integer::valueOf)
-                    .map(LottoNumber::new)
-                    .collect(Collectors.toList())), new LottoNumber(bonusNumber) );
+        return new WinningNumbers(convertStringToLottoNumbers(numbers),
+                new LottoNumber(bonusNumber));
+    }
+
+    private LottoNumbers convertStringToLottoNumbers(String[] str) {
+        return new LottoNumbers(Arrays.stream(str)
```

Comment:
프로그래밍 요구사항에 `규칙 5: 줄여쓰지 않는다(축약 금지).`가 있습니다.
내부에서 사용되는 지역 변수라도 축약하지 말고 그 의미를 전달할 수 있게 네이밍 하시면 좋을 것 같아요
```suggestion
        return new LottoNumbers(Arrays.stream(numbers)
```